missions/ME_Caspian_Minor_Missions.txt

```diff
@@ -373,8 +373,13 @@ cpm_missions_5 = {
 		icon = mission_mosque
 		position = 3
 		trigger = {
-			temple = 3
-			religious_unity = 1
+			OR = {
+				temple = 3
+ 				taxation_lvl_2 = 3
+				cathedral = 3
+ 				taxation_lvl_4 = 3
+				religious_unity = 1
```

Comment:
religious_unity = 1 should be outside the OR

Reply:
Thanks, good catch.